test/OpenTelemetry.Tests/Metrics/MetricAPITest.cs

```diff
@@ -129,7 +128,7 @@ void ProcessExport(Batch<Metric> batch)
 
             var metricReader = new BaseExportingMetricReader(metricExporter);
 
-            var meter = new Meter(meterName);
+            var meter = new Meter("TestMeter" + exportDelta);
```

Comment:
I suggest that we use the full qualified class name plus some postfix to avoid conflicts.

Reply:
The conflicts could come from other test classes creating a `Meter` using the same name right?
So would something like this be good enough of should the current method name creating the meter also be taken into account.

```c#
var meter = new Meter(tyepof(this).Fullname + '.' + exportDelta);
```